projects/components/src/stepper/stepper-tab.component.ts

```diff
@@ -0,0 +1,27 @@
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+import { AbstractControl } from '@angular/forms';
+import { IconType } from '@hypertrace/assets-library';
+
+import { ContentHolder, CONTENT_HOLDER_TEMPLATE } from '../content/content-holder';
+
+@Component({
+  selector: `ht-stepper-tab`,
+  template: CONTENT_HOLDER_TEMPLATE,
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class StepperTabComponent extends ContentHolder {
+  @Input()
+  public label!: string;
+
+  @Input()
+  public icon?: string = IconType.Edit;
```

Comment:
I believe that it's not necessary to do the input optional and assign a value by default, should be the same for the next inputs

Reply:
It's mandatory for class fields to be initialized. Either we can use an assertion or mark them optional. Correct me if m wrong.

Reply:
I think the point @palbizu is making here is, that whenever you assign a default value you actually do not need it to be marked optional.
`public icon?: string = IconType.Edit;` can be replaced with `public icon: string = IconType.Edit;`

Reply:
oh, I missed that! @itssharmasandeep thanks! Updated.